x-pack/plugins/apm/typings/elasticsearch/aggregations.ts

```diff
@@ -145,6 +145,15 @@ export interface AggregationOptionsByType {
     >;
     keyed?: boolean;
   } & AggregationSourceOptions;
+  range: {
```

Comment:
We are investigating moving these types into the JS client (see #77720.) No timeframe on that, but worth considering.

I'm always in favor of explicitly declaring dependencies, though in this case we would create a circular one since APM has Task Manager as an optional dependency.

As for the type changes here I'd like to defer to @dgieselaar on that one before approving.

Reply:
Yeah I wasn't crazy about this, but checked with @dgieselaar and he gave a thumbsup.
He said he'd review on the PR 👍